app/src/org/commcare/engine/extensions/IntentExtensionParser.java

```diff
@@ -42,7 +42,8 @@ public void handle(XFormParser p, Element e, Object parent) {
 
         Log.d(TAG, "0123 extention parser appearance is: " + appearance);
 
-        String label = e.getAttributeValue(null, "button-label");
+        String getButtonLabel = e.getAttributeValue(null, "button-label");
```

Comment:
Ah I see, it's like "get" something from the intent. Makes more sense, I think it just comes off a little awkwardly because of the conflict with the standard meaning for `get`. I can't really think of a better option though